### PR TITLE
Accessing webhook info via UI or API (additional info)

### PR DESCRIPTION
**Pull request by karensawrey**

### CODE REVIEW

File: pages/pipelines/incoming_webhooks.md.erb [hunk 1]
```diff
@@ -13,7 +13,26 @@ Secrets are pipeline-specific. A secret can’t prove that it hasn’t been acqu
 
 You can also provide the permitted IPs for GitHub Enterprise Server, BitBucket Server, GitLab Community, and GitLabEnterprise in your [organization's settings](https://buildkite.com/organizations/~/settings). To enable IP filtering for them, in the [Repository providers](https://buildkite.com/organizations/krnswry/repository-providers) tab, enter the allowed IP addresses. 
 
-For non-server BitBucket, IP filtering for webhooks is currently unavailable.
+For non-server BitBucket, IP filtering for webhooks is currently unavailable.
```
Comment (plaindocs) [inline pages/pipelines/incoming_webhooks.md.erb:16]: In the previous paragraph we let a `krnswry` slip through in the URL.

Comment (plaindocs) [reply pages/pipelines/incoming_webhooks.md.erb:16]: Hosted BitBucket maybe? I know that isn't included in this PR, but that read awkwardly.  Alternatively we could go for IP filertering is only availabl for:
* X
* Y
* Z

Comment (plaindocs) [reply pages/pipelines/incoming_webhooks.md.erb:16]: /Also (sorry) these should be H2s right? Maybe with a `{:toc}`

File: pages/pipelines/incoming_webhooks.md.erb [hunk 2]
```diff
@@ -13,7 +13,26 @@ Secrets are pipeline-specific. A secret can’t prove that it hasn’t been acqu
 
 You can also provide the permitted IPs for GitHub Enterprise Server, BitBucket Server, GitLab Community, and GitLabEnterprise in your [organization's settings](https://buildkite.com/organizations/~/settings). To enable IP filtering for them, in the [Repository providers](https://buildkite.com/organizations/krnswry/repository-providers) tab, enter the allowed IP addresses. 
 
-For non-server BitBucket, IP filtering for webhooks is currently unavailable.
+For non-server BitBucket, IP filtering for webhooks is currently unavailable.  
+
+### Can the webhook that triggered a build be accessed in the UI or API?
```
Comment (plaindocs) [inline pages/pipelines/incoming_webhooks.md.erb:18]:
```suggestion
### How do I access the webhook that triggered a build? 
```

Comment (plaindocs) [reply pages/pipelines/incoming_webhooks.md.erb:18]: And the reword to the two options? 

With the first point, an example URL might help.

**Review by plaindocs — COMMENTED** [review]
Seems like a good addition.

File: pages/pipelines/incoming_webhooks.md.erb (hunk 1 above)
Comment (karensawrey) [reply pages/pipelines/incoming_webhooks.md.erb:16]: Implementing all the suggested changes & remaking the Qs into H2s - but I'm a bit afraid that it's going to wreck the formatting soooo hard.

**Comment by plaindocs** [comment]
Direct preview here https://docs-review-pr-1103.herokuapp.com/docs/pipelines/incoming-webhooks

**Review by plaindocs — APPROVED** [review]
Seems fine to me. :+1:

**Pull request merged**